src/components/RoutingSlip/CreateRoutingSlipDetails.vue

```diff
@@ -66,6 +67,9 @@ import { useCreateRoutingSlipDetails } from '@/composables/RoutingSlip'
       isUniqueNumber,
       errorMessage
     } = useCreateRoutingSlipDetails()
+
+    routingSlipDate.value = routingSlipDate.value || moment().format('YYYY-MM-DD')
```

Comment:
can we do it before importing routingSlipDate from useCreateRoutingSlipDetails file?